exercises/yacht/description.md

```diff
@@ -0,0 +1,4 @@
+Score a single throw of dice in *Yacht*.
+
+See the description of [Yacht](https://en.wikipedia.org/wiki/Yacht_(dice_game))  on Wikipedia for the scoring system.
```

Comment:
Minor nit pick.  There is an extra space between the url and the word `on` or perhaps reword:
```json
A description of the Yacht scoring system, used by this exercise, can be found at [Wikipedia](https://en.wikipedia.org/wiki/Yacht_(dice_game)).
```
Alternativly you could incorporate the scoring system into the description.md file.  This file becomes a part of the README that each student receives when they fetch this exercise.  It would be more convenient if the student didn't have to follow a trail of bread crumbs to get to the documentation.  I think this would also serve to protect the exercise from any changes that might crop up on the Wikipedia page that would not be immediately detected at Exercism.

Reply:
The sorted/unsorted tests were added on the basis of "make early tests easy to pass". It is perhaps unnecessary to have sorted/unsorted tests throughout and some can be trimmed.  It is a good idea to copy the scoring system into the description.md file, and I'll do that.